dub.sdl

```diff
@@ -4,8 +4,9 @@ authors "Walter Bright"
 copyright "Copyright © 1999-2017, Digital Mars"
 license "BSL-1.0"
 
-targetType "none"
+targetType "library"
```

Comment:
I was told by Sönke to not do this https://github.com/dlang/dmd/pull/6771#discussion_r116365515.